spikeinterface/core/baserecording.py

```diff
@@ -426,7 +426,19 @@ def set_channel_locations(self, locations, channel_ids=None):
             raise ValueError('set_channel_locations(..) destroy the probe description, prefer set_probes(..)')
         self.set_property('location', locations, ids=channel_ids)
 
-    def get_channel_locations(self, channel_ids=None, locations_2d=True):
+    def get_channel_locations(self, channel_ids=None, dimensions: str='XY'):
```

Comment:
I would use lower case for 'xy' because we alreaady have this in probeinterface.